regparser/web/jobs/utils.py

```diff
@@ -1,7 +1,11 @@
 from regparser.tasks import run_eregs_command
+from django.conf import settings as django_settings
+from django.core.mail import get_connection, send_mail
 import django_rq
 import settings
 
+eregs_site_api_url = getattr(settings, "EREGS_SITE_API_URL")
```

Comment:
Good point, and yes, it would make sense to put a check for it in initialization.